sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/FileAsyncAPITests.groovy

```diff
@@ -969,8 +974,8 @@ class FileAsyncAPITests extends APISpec {
 
         def properties = primaryFileAsyncClient.getProperties().block().getSmbProperties()
 
-        properties.getFileCreationTime() == smbProperties.getFileCreationTime()
-        properties.getFileLastWriteTime() == smbProperties.getFileLastWriteTime()
+        properties.getFileCreationTime().truncatedTo(ChronoUnit.MICROS) == smbProperties.getFileCreationTime().truncatedTo(ChronoUnit.MICROS)
```

Comment:
Should helper APIs be added to compare file creation times instead of copying everywhere?

Reply:
yeah, something in test-shared that can `compareDatesWithPrecision`.